src/test/scala/com/redislabs/provider/redis/sql/SparkSqlStandaloneSuite.scala

```diff
@@ -0,0 +1,110 @@
+package com.redislabs.provider.redis.sql
```

Comment:
I wouldn't separate tests into `sql` and `df` folder since they basically all test dataframes. Also, there are RedisSqlStandaloneSuite, BinaryRedisDataframeSuite and HashRedisDataframeSuite  in `rdd` folder. Should it be moved to `df/sql` folder?